schema.sql

```diff
@@ -1,87 +1,92 @@
-
-
 DROP DATABASE IF EXISTS grassroots;
 
 CREATE DATABASE grassroots;
 
 USE grassroots;
--- ---
--- Globals
--- ---
+
 
 -- SET SQL_MODE="NO_AUTO_VALUE_ON_ZERO";
 -- SET FOREIGN_KEY_CHECKS=0;
 
+
 -- Table 'Races'
 
-DROP TABLE IF EXISTS RACES;
+DROP TABLE IF EXISTS `Races`;
 		
-CREATE TABLE RACES (
-  `id` INTEGER AUTO_INCREMENT,
-  `Date` DATE NULL DEFAULT NULL,
-  `Location` VARCHAR(100) NULL DEFAULT NULL,
+CREATE TABLE `Races` (
+  `id` INTEGER NULL AUTO_INCREMENT DEFAULT NULL,
+  `date` DATE NULL DEFAULT NULL,
+  `location` VARCHAR(50) NULL DEFAULT NULL,
```

Comment:
should we change 'location' to 'city' and 'state' instead of one location? would help in the long run (of this two-week project...)